app/routes/__auth/user.server.ts

```diff
@@ -30,3 +34,37 @@ export async function updateSession(user: any) {
   })
   await storage.commitSession(session)
 }
+
+export async function clearUserToken(sub: string) {
+  const db = await tables()
+  const targetSessionResults = await db.sessions.query({
+    IndexName: 'bySub',
+    KeyConditionExpression: '#sub = :sub',
+    ExpressionAttributeNames: {
+      '#sub': 'sub',
+      '#token': 'token',
+      '#idx': '_idx',
+      '#ttl': '_ttl',
+    },
+    ExpressionAttributeValues: {
+      ':sub': sub,
+    },
+    ProjectionExpression: '#token, #sub, #idx, #ttl',
+  })
+
+  if (!targetSessionResults.Items) return
+
+  const targetSession = targetSessionResults.Items[0]
```

Comment:
Need to loop over all matching sessions.

Reply:
Latest version takes all of items and uses Promise.all to resolve however many there may be